feat: add ethereum-metrics-exporter (#3573)
paradigmxyz/reth: book/installation/docker.md

To run Reth with Docker Compose, run the following command from a shell inside the root directory of this repository:

```bash
./etc/generate-jwt.sh
docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
```

> > **Note:**
> > If you want to run Reth without Lighthouse:
> > - the JWT for the consensus client can be found at path `etc/jwttoken/jwt.hex` of this repository, after the `etc/generate-jwt.sh` script is run
> > - the Reth Engine API is accessible on `localhost:8551`
> > - start Docker Compose without the `etc/lighthouse.yml` file:
> > ```bash
> > ./etc/generate-jwt.sh
> > docker compose -f etc/docker-compose.yml up -d
> > ```

Reviewer: ```suggestion
> **Note**
>
> If you want to run Reth with a CL that is not Lighthouse:
>
> - The JWT for the consensus client can be found at `etc/jwttoken/jwt.hex` in this repository, after the `etc/generate-jwt.sh` script is run
> - The Reth Engine API is accessible on `localhost:8551`
```

docker compose -f etc/docker-compose.yml up -d
```

> If you want to also run Lighthouse, run the following command instead:

Reviewer: Add a note here like

```
> **Note**
>
> If you want to run without Lighthouse, the JWT for the consensus client is in ..., and the engine API is on ...
```

To explain how to get the JWT for your own CL setup (and where reth's engine API is)

```bash
./etc/generate-jwt.sh
docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
```

To check if Reth is running correctly, run:

```bash
docker compose logs -f reth
```

The default `docker-compose.yml` file will create three containers:

- Reth
- Prometheus
- Grafana

The optional `lighthouse.yml` file will create two containers:

- Lighthouse

> - ethereum-metrics-exporter

Reviewer: Add a link here and quote it, i.e. \[\`ethereum-metrics-exporter`\](link)
